### ISSUE TITLE
[Bug] Windows system Error, when trying to edit the default stylesheet

### ISSUE THREAD

**Issue by Golddouble** [alessivs/supermemo-wine#32]
I use your Lutris variant of SM 17. I use the installation, I have made about 1.5 years ago.

When I go to ...
menu->Tools->Optins->Fonts
... press the "stylesheet button" and press the edit button, then I get the following message:

![ksnip_20220110-151805](https://user-images.githubusercontent.com/13977359/148780774-eb60d670-da2d-4312-8637-4dce96ff328b.jpg)

.
.

Lutris: 0.5.7.1
![ksnip_20220110-215526](https://user-images.githubusercontent.com/13977359/148838097-38f7993b-c1fb-4c3f-9f95-22601792d539.jpg)

**Comment by Golddouble** [comment]
I have solved this.

1. open Lutris
2. select the programme (-> Supermemo 17)
3. At the right side click on "Wine Registry"
-> Wine Registry opens
4. navigate to HKEY_LOCAL_MACHINE\SOFTWARE\Classes\CSSfile
5. in the folder "CSSfile" create the key "shell"
6. in the created folder shell create the key "open"
7. in the created folder "open" create the key "command"
8. select the created folder "command"
9. double click on "(standard)" (on right side)
10. put in the filed "C:\windows\system32\notepad.exe %1"
11. close the registry
12. start supermemo 17 new and try if it now works.
-> result: now we can edit the stylesheet

Could this "registry entry" be included in your Lutris SM Installer, so that is made automatically in the future?

**Comment by alessivs** [comment]
> Could this "registry entry" be included in your Lutris SM Installer, so that is made automatically in the future?

Yes. This is the correct way, for CSS and other file types. It will be included.

**Comment by Golddouble** [comment]
I get a similar error, when I try to make a SM repair.

File-> Repair Collection "Basic Checkup"

Result:
![k20220209-135730](https://user-images.githubusercontent.com/13977359/153205756-0bbae27a-61a4-4cd8-a783-48a09ea1e514.jpg)

But here the entry in the registry is already present.
![k20220209-135958](https://user-images.githubusercontent.com/13977359/153206094-daa3ea08-3f8b-47fb-ae68-a6dc5d48e678.jpg)

 So my expectation would be, that it should work. Strange. Any idea?
